vllm/v1/core/sched/scheduler.py

```diff
@@ -100,6 +100,8 @@ def __init__(
 
         # P/D: requests in process of recving KV transfers
         self.finished_recving_kv_req_ids: set[str] = set()
+        # P/D: completed requests with unfreed KV blocks
+        self.unfreed_kv_req_ids: set[str] = set()
```

Comment:
WDYT about calling it something different like `pending_kv_free`?

Reply:
thats good to me